internal/test/test.go

```diff
@@ -48,6 +53,12 @@ func (suite *BaseSuite) SetupSuite() {
 	if err := godotenv.Load("../.env"); err != nil {
 		suite.T().Log(err)
 	}
+	if os.Getenv("TRACING") == "1" {
+		_, err := initTracing()
+		if err != nil {
+			suite.FailNow("failed to initialize tracing %s", err)
+		}
+	}
```

Comment:
sets up exporting to jaeger when env TRACING=1. see docker-compose env variable for export configuration variables